src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -746,4 +746,10 @@
     <value>Suggestions found: {0}</value>
     <comment>{0} will be replaced with a number.</comment>
   </data>
-</root>
+  <data name="NewTabAnnouncement" xml:space="preserve">
+    <value>{} profile tab created successfully</value>
+  </data>
+  <data name="SplitPaneAnnouncement" xml:space="preserve">
+    <value>{} profile pane created successfully</value>
+  </data>

```

Comment:
I'm not sure whether this will work well after localization.
Maybe something like this instead?
> A pane for "{}" was created

Reply:
Went for `A new tab for the "{}" profile was created successfully`.
- Kept the "" in case the screen reader would get confused with the grammar
- I like the idea of keeping the "new" and "successfully", but idk if it's strictly necessary. Open to changing it if people complain it might be too verbose haha.